### ISSUE TITLE
Error handling on workflow without status

### ISSUE THREAD

**Issue by HeartlandTechie** [heloufir/filament-workflow-manager#10]
Attempt to read property "status" on null at /var/www/wbl/vendor/heloufir/filament-workflow-manager/src/Core/WorkflowResource.php

I believe this occurs when you add the package to an existing table with models without a status.  If the status doesn't exist, it should be created using the first status for the workflow rather than an error.

**Comment by SlavaSt295** [comment]
have the similar issue [code share](https://flareapp.io/share/dmk1koY7#F127)

**Comment by SlavaSt295** [comment]
> 
i guess you miss this part of the instruction https://github.com/heloufir/filament-workflow-manager/blob/master/QUICKSTART.md#permissions

**Comment by HeartlandTechie** [comment]
Looking at the code sample you just shared elsewhere, you are actually running into the same problem - if the model isn't created with the workflow already attached, it's null and that blows up the code.  I think there should be a sanity check to make the workflow default to whatever the first status is if there isn't a record for that model - instead it grabs nothing and gives the error on the null pluck.

**Comment by aleksanm** [comment]
Yes. It's true for me too, on existing models, there is no status and all breaks.

**Comment by heloufir** [comment]
Hello,
I will try to check this ASAP and let u know, sorry for the late answer.
